pkg/runctl/config_test.go

```diff
@@ -74,6 +74,10 @@ func Test_loadPipelineRunsConfig_EmptyEntries(t *testing.T) {
 	assert.DeepEqual(t, expectedConfig, resultConfig)
 }
 
+var metav1DurationMinutes = func(d time.Duration) *metav1.Duration {
+	return &metav1.Duration{Duration: time.Duration(time.Minute * d)}
+}
+
```

Comment:
Does it compile?

I would not prescribe the time unit, but take any `time.Duration`as parameter.